lib/cyclonedx/report_ruby_gems_cyclonedx.rb

```diff
@@ -6,5 +6,17 @@ class ReportRubyGems < Base
     def initialize(scan_report, config = {})
       super(scan_report, config)
     end
+
+    def build_component(dependency)
+      super.merge({
```

Comment:
Probably a stupid question. But what does `super.merge` do here?
I don't see `merge` defined in the super class.

Reply:
Good question. This hash [merge](https://apidock.com/ruby/Hash/merge) of ruby. The `build_component` method of superclass returns a hash. In the Cyclonedx::ReportRubyGems, we are merging license information. This is done this way, because we will have license information available to Ruby gems for now.